substrate/frame/revive/src/evm/runtime.rs

```diff
@@ -55,7 +55,7 @@ type CallOf<T> = <T as frame_system::Config>::RuntimeCall;
 /// - Not too high, ensuring the gas value is large enough (at least 7 digits) to encode the
 ///   ref_time, proof_size, and deposit into the less significant (6 lower) digits of the gas value.
 /// - Not too low, enabling users to adjust the gas price to define a tip.
-pub const GAS_PRICE: u32 = 1_000u32;
+pub(crate) const GAS_PRICE: u64 = 1_000u64;
```

Comment:
This might be interesting to access for downstream consumers, why is it no longer public?

Reply:
Nevermind, it doesn't matter because the getter is public.